js/src/run_trees.ts

```diff
@@ -28,6 +28,8 @@ export function convertToDottedOrderFormat(
   );
 }
 
+export const _LC_CONTEXT_VARIABLES_KEY = Symbol.for("lc:context_variables");
```

Comment:
_nit_: do we want to export this symbol out? Ideally would avoid exporting these internals out. If no-option is available, maybe `@hidden` jsdoc tag would help (saw that tag somewhere when reading LC.js)

Reply:
Yeah can refactor

Reply:
Done